enigma/src/main/java/cuchaz/enigma/translation/mapping/serde/srg/SrgMappingsWriter.java

```diff
@@ -44,9 +43,9 @@ public void write(EntryTree<EntryMapping> mappings, MappingDelta<EntryMapping> d
 		List<String> fieldLines = new ArrayList<>();
 		List<String> methodLines = new ArrayList<>();
 
-		Collection<Entry<?>> rootEntries = Lists.newArrayList(mappings).stream()
+		List<? extends Entry<?>> rootEntries = Lists.newArrayList(mappings).stream()
```

Comment:
Can just use `var` if the type is too complex and can be inferred

Reply:
It's not too complex imo